templates/settings.json.epp

```diff
@@ -54,19 +56,19 @@
   //the default text of a pad
   "defaultPadText" : "<%= $etherpad::default_pad_text -%>",
 
-  /* Default Pad behavior, users can override by changing */
+  //Pad behavior
   "padOptions": {
-    "noColors": false,
-    "showControls": true,
-    "showChat": true,
-    "showLineNumbers": true,
-    "useMonospaceFont": false,
-    "userName": false,
-    "userColor": false,
-    "rtl": false,
-    "alwaysShowChat": false,
-    "chatAndUsers": false,
-    "lang": "en-gb"
+    "noColors": <%= $etherpad::_real_padoptions['nocolors'] -%>,
+    "showControls": <%= $etherpad::_real_padoptions['showcontrols'] -%>,
+    "showChat": <%= $etherpad::_real_padoptions['showchat'] -%>,
+    "showLineNumbers": <%= $etherpad::_real_padoptions['showlinenumbers'] -%>,
+    "useMonospaceFont": <%= $etherpad::_real_padoptions['usemonospacefont'] -%>,
+    "userName": <%= $etherpad::_real_padoptions['username'] -%>,
+    "userColor": <%= $etherpad::_real_padoptions['usercolor'] -%>,
+    "rtl": <%= $etherpad::_real_padoptions['rtl'] -%>,
+    "alwaysShowChat": <%= $etherpad::_real_padoptions['alwaysshowchat'] -%>,
+    "chatAndUsers": <%= $etherpad::_real_padoptions['chatandusers'] -%>,
+    "lang": "<%= $etherpad::_real_padoptions['lang'] -%>"
```

Comment:
I wonder if you can get rid of most of this somehow.  Maybe by using https://github.com/puppetlabs/puppetlabs-stdlib#to_json_pretty in the code?  (You'd have to use camelcase in for your puppet hash keys, but this isn't a problem.)

Reply:
Sorry, but I don't understand what did you say about this. Can you explain me what do you want for this part of code?

Reply:
It's not a blocker, but personally, I might have tried to remove the amount of repetition in the template code.  You could do this by either using some ruby function in the template, or using a puppet function in the manifest to generate a complete string of json and interpolate that in the template instead.

Reply:
Ok, thanks.